Fleece/Fleece.fs

```diff
@@ -210,8 +210,87 @@ module SystemJson =
     let inline JString (x: string) = if isNull x then JNull else JsonPrimitive x :> JsonValue
     let inline JNumber (x: decimal) = JsonPrimitive x :> JsonValue
 
-    #endif
+#endif
+
+#if SYSTEMTEXTJSON
+
+module SystemTextJson =
+
+    open System.Text.Json
+
+    type JsonValueW=Utf8JsonWriter -> string option-> unit
+    type JsonObjectW=Map<string, JsonValueW>

```

Comment:
I think Json allows more than one element with the same key, so we need a MultiMap or simply an array.